components/core/src/clp/streaming_archive/reader/Message.hpp

```diff
@@ -26,7 +27,8 @@ class Message {
     friend class Archive;
 
     // Variables
-    size_t m_message_number;
+    size_t m_ix_in_split;
```

Comment:
Without context, `split` is probably ambiguous. So perhaps `m_ix_in_file_split` is a better name.

Reply:
You also need to rename `get_ix_in_split`, right?